src/utils/printSchema.ts

```diff
@@ -0,0 +1 @@
+export const defaultPrintSchemaOptions: PrintSchemaOptions = { commentDescriptions: false };
```

Comment:
`PrintSchemaOptions` isn't imported and causes the transpilation to fail, just fyi

Reply:
Yup, forgot to remove this file from this PR.  I'll rebase everything later.  Removed in the last commit.